apps/nouns-camp/src/components/activity-feed.js

```diff
@@ -272,8 +272,19 @@ const ItemTitle = ({ item, context }) => {
   );
 
   switch (item.type) {
-    case "signature":
-      return accountName;
+    case "signature": {
```

Comment:
I replaced this with `"candidate-signature-added"` further down, meant to remove but forgot. Are we still creating `type: "signature"` items somewhere? 🤔